### PR TITLE
Multi wallet order

### PR DESCRIPTION
**Pull request by rmisio**
Implements multi-wallet related functionality in the Order Transactions page and Order Detail page. With this piece and #1588, the full order lifecycle should be in play (from listing creation to order resolution).

### CODE REVIEW

File: js/models/BaseModel.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default class extends Model {
     // todo: will it break things if we unset a nested attribute?
     if (!opts.unset) {
       // let's work off of a clone since we modify attrs
-      attrs = JSON.parse(JSON.stringify(attrs));
+      // attrs = JSON.parse(JSON.stringify(attrs));
```
Comment (jjeffryes) [inline js/models/BaseModel.js]: It looks like taking this out is what breaks the moderatorInfo save in the settings.

File: js/models/order/Order.js [hunk 2]
```diff
@@ -129,31 +173,33 @@ export default class extends BaseOrder {
...
+      response.rawContract = this.rawResponse.contract;
+
+      const payment = response.contract.buyerOrder.payment;
 
       // convert price fields
-      response.contract.buyerOrder.payment.amount =
-        integerToDecimal(response.contract.buyerOrder.payment.amount,
-          app.serverConfig.cryptoCurrency);
+      payment.amount =
+        integerToDecimal(payment.amount, payment.coin);
```
[the remark this answers is not in the document]
Comment (rmisio) [reply js/models/order/Order.js]: Well this is what we do in all models that have prices in integer/base-unit format. We convert them to decimal/non-base-unit for display purposes.

[the remark this answers is not in the document]
Comment (rmisio) [reply js/models/order/Order.js]: So the converted amount is used, anywhere it's displayed. It's just doesn't happen to be used in parse().

**Comment by jjeffryes** [comment]
@rmisio is this ready to merge? If so you can or I can.

**Pull request merged**